4337/package.json

```diff
@@ -46,31 +46,31 @@
   },
   "devDependencies": {
     "@account-abstraction/contracts": "^0.6.0",
-    "@nomicfoundation/hardhat-toolbox": "^3.0.0",
+    "@nomicfoundation/hardhat-toolbox": "^4.0.0",
     "@openzeppelin/contracts": "^5.0.0",
-    "@types/chai": "^4.3.1",
-    "@types/mocha": "^9.1.1",
-    "@types/node": "^18.18.6",
-    "@types/yargs": "^17.0.10",
-    "@typescript-eslint/eslint-plugin": "^5.28.0",
-    "@typescript-eslint/parser": "^5.28.0",
+    "@types/chai": "^4.3.11",
+    "@types/mocha": "^10.0.6",
+    "@types/node": "^20.9.4",
```

Comment:
We should update the CI to run on Node 20 instead of 18 then (so the types version matches what we actually use):

https://github.com/safe-global/safe-modules/blob/7cfb9ddeb5f4c741987a360477928d1c581f22cd/.github/workflows/ci_4337.yml#L14